src/elements/_form.pug

```diff
@@ -24,7 +24,7 @@ mixin input(type, label, name, id, datatype, datasize)
     else
       .input-group
         input(type=type, id=id, name=name)&attributes(attributes)
-        label(for=id)= label
+        label(for=id)&attributes(attributes)= label
```

Comment:
Input and label will have the same attribute,

Reply:
Did we discuss passing a class to the into the mixin which would be applied to the label only?

Reply:
Wasn't supposed to be added, removing this.

Reply:
I remember James you were going to make amends here